src/animation.js

```diff
@@ -33,8 +33,8 @@ var _cancelFrame = function(inId) {
 };
 
 for (var i = 0, pl = prefix.length, p, wc, wr; (p = prefix[i]) || i < pl; i++) {
-	// if we're on ios 6 just use setTimeout, requestAnimationFrame has some kinks currently
-	if (platform.ios >= 6) {
+	// if we're on ios 6 just use setTimeout, requestAnimationFrame has some kinks
+	if (platform.ios < 7) {
```

Comment:
Interesting - for iOS 5 and below, it previously fell back to `setTimeout`, as well, because the conditional on [L45](https://github.com/enyojs/enyo/pull/1373/files#diff-2314e4f2e74b0a130fa863211c157653L45) failed. Wonder if we should let these iOS versions run through this feature detection logic, in case something changes in the future, and only make a special case for iOS 6. Otherwise, we should update the comment to be clearer about our intention here.